type-tests/files/createSlice.typetest.ts

```diff
@@ -312,3 +314,57 @@ function expectType<T>(t: T) {
     }
   })
 }
+
+/** Test: wrapping createSlice should be possible */
```

Comment:
Nice to see that A) this is possible, and B) we have a test for it.

Reply:
Also, can we add this example to the "Usage with TS" docs page?

Reply:
Honestly, without these tests I wouldn't have the confidence to change on single letter in the typings. Those are so invaluable!